src/components/Main/utils.js

```diff
@@ -0,0 +1,21 @@
+import {
+  FISCAL_YEAR_ROUTE,
+  FUNDS_ROUTE,
+  GROUPS_ROUTE,
+  LEDGERS_ROUTE,
+} from '../../common/const';
+
+const viewPermissionsMap = {
+  'ui-finance.ledger.view': LEDGERS_ROUTE,
+  'ui-finance.fiscal-year.view': FISCAL_YEAR_ROUTE,
+  'ui-finance.group.view': GROUPS_ROUTE,
+  'ui-finance.fund-budget.view': FUNDS_ROUTE,
+};
+
+export const getInitialRoute = (stripes) => {
```

Comment:
do you need the whole stripes here?

Reply:
maybe it makes sense to pass here only what you need, then you can use some usememo above